baseframe/__init__.py

```diff
@@ -194,6 +195,16 @@ def init_app(
                 ],
             )
 
+        # Setup secret key rotation
+        if app.config['SECRET_KEY']:  # Always present as it's a default
+            if not app.config.get('SECRET_KEYS'):
+                app.config['SECRET_KEYS'] = [app.config['SECRET_KEY']]
+        elif 'SECRET_KEYS' in app.config:
+            app.config['SECRET_KEY'] = app.config['SECRET_KEYS'][0]
```

Comment:
This will fail and throw a confusing error, if `SECRET_KEYS = []` and `SECRET_KEY = ''` for some reason. It shouldn't happen, just mentioning.

Reply:
Changed to check for truthiness, and also log a DEBUG explaining which var is being set from which.